pkg/http/api.go

```diff
@@ -205,6 +205,17 @@ func (a *api) constructActorEndpoints() []Endpoint {
 	}
 }
 
+func (a *api) constructTransactionEndpoints() []Endpoint {
```

Comment:
Thanks for the contribution. I would suggest following changes:
1. This is a state operation, so it should be state/transaction (After the multi store [PR](https://github.com/dapr/dapr/pull/912/files) is merged, it would be state/<storename>/transaction)
2. The functionality should be exposed over grpc (added in pkg/grpc) as well, 
3. For gRPC, proto file needs to be updated (/pkg/proto/dapr/dapr.proto)
4. The api should return error when called for a non transactional state store.
5. The code should be refactored, it should not call into pkg/actors.go for making state changes as actors modify the keys for storage, Actors pkg deals with actor specific logic, so making the state calls go there for transaction is not good from code organization, maintainability perspective.

     -

Reply:
Great!, Thanks for making the changes. Since the last review, Multi state store changes have been merged. Now following changes would also be needed:
1. The url would be state/<strorename>/transaction.

Also please incorporate the changes to expose it over gRPC>